packages/flutter/test/material/buttons_test.dart

```diff
@@ -616,141 +616,4 @@ void main() {
         paintsExactlyCountTimes(#clipPath, 0),
     );
   });
-
-  testWidgets('MaterialButton shape overrides ButtonTheme shape', (WidgetTester tester) async {
-    // Regression test for https://github.com/flutter/flutter/issues/29146
-    await tester.pumpWidget(
-      Directionality(
-        textDirection: TextDirection.ltr,
-        child: MaterialButton(
-          onPressed: () { },
-          shape: const StadiumBorder(),
-          child: const Text('button'),
-        ),
-      ),
-    );
-
-    final Finder rawButtonMaterial = find.descendant(
-      of: find.byType(MaterialButton),
-      matching: find.byType(Material),
-    );
-    expect(tester.widget<Material>(rawButtonMaterial).shape, const StadiumBorder());
-  });
-
-  testWidgets('MaterialButton disabled default is correct.', (WidgetTester tester) async {
-    // Regression test for https://github.com/flutter/flutter/issues/30012.
-
-    final Finder rawButtonMaterial = find.descendant(
-      of: find.byType(MaterialButton),
-      matching: find.byType(Material),
-    );
-
-    await tester.pumpWidget(
-      const Directionality(
-        textDirection: TextDirection.ltr,
-        child: MaterialButton(
-          disabledColor: Color(0xff00ff00),
-          onPressed: null,
-          child: Text('button'),
-        ),
-      ),
-    );
-
-    final Material material = tester.widget<Material>(rawButtonMaterial);
-    expect(material.color, const Color(0xff00ff00));
-  });
-
-  group('feedback', () {
-    FeedbackTester feedback;
```

Comment:
@shihaohong  Why did you remove those tests?

Reply:
It was causing a merge conflict since it's part of an ongoing effort to remove the `buttons_test.dart` file. There are more specific files that some of these tests belong to (ie. `flat_button_test.dart`), and it made more sense to have them there.

However, if I seem to have removed a test accidentally that wasn't intentional, please let me know.